apps/feedahelp-cms/schemas/ingredient.ts

```diff
@@ -0,0 +1,47 @@
+import {LemonIcon} from '@sanity/icons'
+import {defineField, defineType} from 'sanity'
+
+export default defineType({
+  name: 'ingredient',
+  title: 'Ingredient',
+  type: 'document',
+  icon: LemonIcon,
+  fields: [
+    defineField({
+      name: 'ingredientName',
+      title: 'Ingredient Name',
+      type: 'string',
+      validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+      name: 'ingredientNameBangla',
+      title: 'Ingredient Name Bangla',
+      type: 'string',
+      validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+        name: 'ingredientImage',
+        title: 'Ingredient Image',
```

Comment:
just `Image`, you see we are already in Ingredients, you don't have name everything ingredients

Reply:
resolved